src/main/java/uk/gov/hmcts/ccd/config/SwaggerConfiguration.java

```diff
@@ -53,6 +54,7 @@ public Docket apiV2() {
             .groupName("v2")
             .select()
             .apis(RequestHandlerSelectors.basePackage(CaseController.class.getPackage().getName()))
+            .apis(RequestHandlerSelectors.basePackage(StartTriggerController.class.getPackage().getName()))
```

Comment:
This is not needed as this is per package, and the package of `StartTriggerController.class` is the same as `CaseController.class`.